sdk/storage/azdatalake/directory/client.go

```diff
@@ -56,6 +56,11 @@ func NewClient(directoryURL string, cred azcore.TokenCredential, options *Client
 	if options == nil {
 		options = &ClientOptions{}
 	}
+	perCallPolicies := []policy.Policy{shared.NewIncludeBlobResponsePolicy()}
```

Comment:
I believe this should go in the `plOpts.PerCall` field instead.

Reply:
I dont think I have access to pipeline options here. It's in the internal package.

Reply:
See above, line 47 where the auth policy is added to `PerRetry`.

Generally speaking, SDK authors should put their policies in `runtime.PipelineOptions`, leaving `ClientOptions` for calling code.

Reply:
the plOpts field above is for the datalake client. We need the custom blob response policy in the blob client code (which we are constructing as a caller)

Reply:
Oh yes I see now, makes sense.